wallet-api/src/Wallet/UTXO/Runtime.hs

```diff
@@ -111,7 +113,33 @@ instance (TypeablePlc a, LiftPlc a) => LiftPlc (Signed a)
 -- | Ada value
 --
 -- TODO: Use [[Wallet.UTXO.Types.Value]] when Integer is supported
-type Value = Int
+data Value = Value Int
```

Comment:
Wouldn't it be nicer to use

```
data Value = Value { getValue :: Int }
```

to not have these `let Value v = ... in v` projections in the contracts (but use `getValue`)? (In fact, why not a `newtype`?)

Reply:
> to not have these let Value v = ... in v projections in the contracts

In the contracts we still have to use pattern matching (or `RecordWildCards`) because the record selectors aren't available when using the plugin (last I checked)

> why not a `newtype`?

That doesn't work because the core-to-plc plugin cannot handle the resulting Core (see https://iohk.myjetbrains.com/youtrack/issue/CGP-414 - this was the reason for the type synonym in the first place)

Reply:
Record selectors indeed don't work, they're a bit non-trivial. Realised I didn't have an issue for this yet, here it is: https://iohk.myjetbrains.com/youtrack/issue/CGP-423

Reply:
Ok, fair enough. 

For the Plutus Playground documentation, we need to make sure to collect all these restrictions in one place, so it is easy for people to see what they can use and what not.

Reply:
I already have https://iohk.myjetbrains.com/youtrack/issue/CGP-422. Although I was hoping I'd just fix most of them ;

Reply:
This is a secondary consideration, but `data Value = Value { getValue :: Int }` is a bit more painful to work with when we export to PureScriptland.

(That's the exporter's fault, not PureScript's, but it's still painful.)